packages/devp2p/package.json

```diff
@@ -62,7 +62,7 @@
     "@scure/base": "1.1.1",
     "debug": "^4.3.3",
     "ethereum-cryptography": "^2.1.2",
-    "lru-cache": "^7.18.3",
+    "lru-cache": "^10.0.0",
```

Comment:
Without having checked if this is the case or not, it would be good if all out lru-cache deps would be Version aligned throughout the packages.

Reply:
10.0.0 is the correct version to match everything else in our usage so we should be good here.  Some of our dev deps use older versions but that shouldn't impact us.

Reply:
Ah ok, but it (generally) wouldn't hurt to align the dev dep versions too here (also just a mention)